extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -22,10 +22,21 @@ import {setStyles} from '../../../src/style';
 import {addParamsToUrl} from '../../../src/url';
 import {timerFor} from '../../../src/timer';
 import {isObject} from '../../../src/types';
+import {VideoEvents} from '../../../src/video-interface';
+import {videoManagerForDoc} from '../../../src/video-manager';
 
 /** @type {number} Value of YouTube player state when playing. */
 const YT_PLAYER_STATE_PLAYING = 1;
 
+/** @type {number} Value of YouTube player state when paused. */
+const YT_PLAYER_STATE_PAUSED = 2;
```

Comment:
This and `YT_PLAYER_STATE_PLAYING` might be better in an enum, to separate them from `YT_PLAYER_FLAG_HIDE_ANNOTATION`.